packages/loader/index.js

```diff
@@ -3,10 +3,10 @@ const mdx = require('@mdx-js/mdx')
 
 const DEFAULT_RENDERER = `
 import React from 'react'
-import { mdx } from '@mdx-js/react'
+import {mdx} from '@mdx-js/react'
 `
 
-const loader = async function (content) {
+module.exports = async function (content) {
```

Comment:
It seems odd to mix ESM and CJS.
Would is make sense to have this be `export default`?
or alternatively to make the `import`s into `require`s?

Reply:
The ESM is for the generated MDX file, which can only be loaded in Babel as it currently includes JSX. The CJS is for the webpack loader.

Most packages in this project are written in CJS and do not require building / bundling, except for Vue and Preact

Reply:
Ah, I missed the template string :sweat_smile: 
My bad, thanks for clarifying.